src/entries/awesome-cheab-quotes.ts

```diff
@@ -9,4 +10,5 @@ export default {
   topic: AcademicTopics.Vue,
   githubUrl: "https://github.com/narze/awesome-cheab-quotes",
   imagePath: undefined,
+  ButtoDisplayoption: ButtoDisplayoption.Seemore,
```

Comment:
Instead of passing Seemore option in every entry, it's better to make ButtonDisplayOption has "See more" as default value. If the entry specify ButtonDisplayOption it would change the label

Also I think just `buttonLabel` (with camelCase) is clear enough.

Reply:
agreed, Will change all.